packages/test262-runner/test262/test/built-ins/Compartment/prototype/Symbol.toStringTag-lockdown.js

```diff
@@ -0,0 +1,13 @@
+/*---
+description:
+includes: [propertyHelper.js]
+features: [Symbol.toStringTag,Compartment,lockdown,ses-xs-parity]
+---*/
+
+lockdown();
+
+assert.sameValue(Compartment.prototype[Symbol.toStringTag], 'Compartment');
+
+verifyNotEnumerable(Compartment.prototype, Symbol.toStringTag);
+verifyNotWritable(Compartment.prototype, Symbol.toStringTag);
+verifyNotConfigurable(Compartment.prototype, Symbol.toStringTag);
```

Comment:
This is a new test262 style test for the post-lockdown appearance of Compartment.prototype.@@toStringTag.